lib/streamlit/elements/alert.py

```diff
@@ -0,0 +1,76 @@
+from streamlit.proto import Alert_pb2
```

Comment:
I would prefer `from streamlit.proto.Alert_pb2 import Alert`. It looks a lot cleaner to me.

Reply:
**UPDATE** Austin and I decided to go with the following approach:
```python
from streamlit.proto.Alert_pb2 import Alert as AlertProto
```